shared/src/packages/articles/libs/types/article-entity.type.ts

```diff
@@ -6,6 +8,7 @@ type ArticleEntityType = {
   promptId: number | null;
   genreId: number | null;
   publishedAt: string | null;
+  userDetails?: UserDetailsResponseDto;
```

Comment:
If it varies upon different requests - then create a separate type for it (that will extend this base type)